### PR TITLE
V2 Adds UI colors for P1 Compatibility

### PR DESCRIPTION
**Pull request by isralduke**
This PR:

- adds two colors present in Pelican 1
- updates MD file for Colors

### CODE REVIEW

File: scss/_badges.scss [hunk 1]
```diff
@@ -26,4 +26,10 @@
   &-ui {
     background-color: $ui;
   }
+  &-ui-light {
+    background-color: $ui-light;
+  }
```
Comment (frogpad7) [inline scss/_badges.scss:31]: This should be the same as the light contrast colors and also have the text be dark.
```suggestion
  &-ui-light {
    background-color: $ui-light;
    color: $black;
  }
```

File: docs/foundation/ui-colors.md [hunk 2]
```diff
@@ -26,6 +26,26 @@ Pelican provides more clarity in the user interface by using a color for UI elem
...
+      <div class="card-body">
+        <p class="mb-0 fw-bold">UI Priority</p>
+        <p class="mb-0">Hex Code: <code>#004ba1</code></p>
+        <p class="mb-0">SCSS Var: <code>$info-900</code></p>
+      </div>
+    </div>
+  </div>
 </div>
 
 Like [Brand Colors](/foundation/status-colors) and [Status Colors](/foundation/status-colors), this can be changed. Change with great care as this has been chosen in accordance with usability and [Color Contrast](/accessibility/color-contrast/) guidelines.
```
Comment (frogpad7) [inline docs/foundation/ui-colors.md:51]: Are the other blocks of text for "Using UI Colors" missing for the new UI shades?

**Review by frogpad7 — CHANGES_REQUESTED** [review]
Pretty sure the UI Colors page in Foundations has some things missing from version 1.

File: docs/foundation/ui-colors.md (hunk 2 above)
Comment (isralduke) [reply docs/foundation/ui-colors.md:51]: it was missing before this PR

**Review by isralduke — COMMENTED** [review]
Comments added @frogpad7

**Review by frogpad7 — APPROVED** [review]
Everything works well on the UI Colors page; not sure why we don't add back in the text explanations of the new colors now that they exist again.

**Pull request merged**